wfe2/wfe.go

```diff
@@ -2479,6 +2479,10 @@ func (wfe *WebFrontEndImpl) GetOrder(ctx context.Context, logEvent *web.RequestE
 		response.Header().Set(headerRetryAfter, strconv.Itoa(orderRetryAfter))
 	}
 
+	orderURL := web.RelativeEndpoint(request,
+		fmt.Sprintf("%s%d/%d", orderPath, acctID, order.Id))
```

Comment:
I know we're doing this elsewhere, but we should consider using path.Join to make this less resistant to future updates wrecking the URLs we return.

Reply:
Filed https://github.com/letsencrypt/boulder/issues/8204 for this